router/src/http/server.rs

```diff
@@ -1101,6 +1219,8 @@ pub async fn run(
         .route("/tokenize", post(tokenize))
         // OpenAI compat route
         .route("/embeddings", post(openai_embed))
+        // Vertex compat route
+        .route("/vertex", post(vertex_compatibility))
```

Comment:
This should not be needed since we only need the endpoint when /env is set.

Reply:
in the TGI case we also always include the endpoint and only conditionally add the api docs. Should we avoid adding it in both cases if the feature is not on?